pygfx/objects/_lights.py

```diff
@@ -0,0 +1,68 @@
+from pygfx.linalg.vector3 import Vector3
+from ._base import WorldObject
+from ..utils.color import Color
+
+
+class Light(WorldObject):
+    """A light object."""
+
+    uniform_type = dict(
+        color="4xf4",
+        intensity="f4",
+    )
+
+    def __init__(self, color=(1, 1, 1, 1), intensity=1):
+        super().__init__()
+        self.color = color
+        self.intensity = intensity  # not used for now
+
+    @property
+    def color(self):
+        return Color(self.uniform_buffer.data["color"])
+
+    @color.setter
+    def color(self, color):
+        color = Color(color)
+        self.uniform_buffer.data["color"] = color
+        self.uniform_buffer.update_range(0, 1)
+
+
+class PointLight(Light):
+    """A light that gets emitted from a single point in all directions."""
+
+    uniform_type = dict(
+        distance="f4",
+        decay="f4",
+    )
+
+    def __init__(self, color=(1, 1, 1, 1), intensity=1, distance=0, decay=1):
+        super().__init__(color, intensity)
+
+        # not used for now
+        self.distance = distance
+        self.decay = decay
```

Comment:
If they are not used, just remove the parameters. They can be added back in later when we do support them.